src/components/pages/public/MovieDetailPage.vue

```diff
@@ -58,10 +58,14 @@ export default {
     this.movieDetails = data;
     this.setMovieTitle(this.movieDetails.title);
   },
+  computed: {
+    metaTitle() {
+      return this.movieDetails ? this.movieDetails.title : "montecinema";
```

Comment:
Pretty much what I was talking about, but you need to rethink the condition because `this.movieDetails` is always truthy.